From 08cefc7ad4353d51719087ef15b09c2c808c5e0c Mon Sep 17 00:00:00 2001
From: Timple Raj M <quic_timple@quicinc.com>
Date: Thu, 4 May 2023 12:43:31 +0530
Subject: [PATCH 210/281] arm: dts: ipq9574: Add build support for various RDPs

This patch adds build support for RDP418, RDP443,
RDP449, RDP453, RDP454, RDP455, RDP456, RDP457,
RDP458, RDP459, RDP467 and RDP469

Change-Id: Ib7c78bf5046a3b7452e86ee01ca2010c39bbaa64
Signed-off-by: Timple Raj M <quic_timple@quicinc.com>
---
 arch/arm/boot/dts/Makefile | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 6aa7dc4db2fc..6d8674a07119 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -1048,6 +1048,18 @@ dtb-$(CONFIG_ARCH_QCOM) += \
 	qcom-ipq4019-ap.dk07.1-c2.dtb \
 	qcom-ipq8064-ap148.dtb \
 	qcom-ipq8064-rb3011.dtb \
+	ipq9574-rdp418.dtb \
+	ipq9574-rdp433.dtb \
+	ipq9574-rdp449.dtb \
+	ipq9574-rdp453.dtb \
+	ipq9574-rdp454.dtb \
+	ipq9574-rdp455.dtb \
+	ipq9574-rdp456.dtb \
+	ipq9574-rdp457.dtb \
+	ipq9574-rdp458.dtb \
+	ipq9574-rdp459.dtb \
+	ipq9574-rdp467.dtb \
+	ipq9574-rdp469.dtb \
 	qcom-msm8226-samsung-s3ve3g.dtb \
 	qcom-msm8660-surf.dtb \
 	qcom-msm8916-samsung-e5.dtb \
-- 
2.17.1

